cmd/cmaf-ingest-receiver/app/channel.go

```diff
@@ -180,7 +180,7 @@ func (ch *channel) addInitDataAndUpdateTimescale(stream stream, init *mp4.InitSe
 
 	ch.addTrData(r)
 	switch sampleEntry {
-	case "avc1", "hvc1", "mp4a", "stpp", "wvtt":
+	case "avc1", "hvc1", "mp4a", "stpp", "wvtt", "ac-3", "ec-3":
 		// OK
```

Comment:
Yes, this should be enough since the receiver doesn't do any fancy stuff to the audio content like changing timescale or re-fragmenting it into other durations.

Reply:
Thanks. Put a TODO.